code/modules/mob/mob_movement.dm

```diff
@@ -140,7 +145,7 @@
 	. = ..()
 
 	if((direct & (direct - 1)) && mob.loc == n) //moved diagonally successfully
-		add_delay *= 2
+		add_delay *= sqrt(2)
```

Comment:
I heavily dislike euclidean movement and think it feels very strange. I will cut the middle at 1.7-1.8x, but 1.41 is too strong for drifting around corners.